server.js

```diff
@@ -1,11 +1,14 @@
 var express = require('express');
 var request = require('request');
 var fs = require('fs');
-var secretString = "9t6aHMrAauERxkR";
+var models = require("./models"); // Database models
 var sessions = require("client-sessions");
 
 var app = express();
 
+// Secret string for sessions cookie
+var secretString = "9t6aHMrAauERxkR";
```

Comment:
Maybe add this to the config/api_keys.json file, or at least in its own config file